core/Debian/stretch/Dockerfile

```diff
@@ -21,3 +22,6 @@ RUN apt-get update \
     && mkdir -p /etc/ansible \
     && echo "[local]" > /etc/ansible/hosts \
     && echo "localhost ansible_connection=local" >> /etc/ansible/hosts
+
+COPY davedaemon /usr/local/bin/
+RUN chmod 755 /usr/local/bin/davedaemon
```

Comment:
I'm not sure how this davedaemon works. 

It seems as though without init there would be no way to test services?

Reply:
Hi,
Very late reply but I have to make some tests to confirm your use case. I also add a similar issue in a recent project where the tested daemon does not want to start. At first I was looking at ansible failing to detect the proper init system to invoke the daemon startup but finish in something which was not so simple so I simply skip this in tests and didn't had time to look further.

However I don't see any case where the process started by the init system can go outside the container. This should never happen as container need to be isolated using cgroups and other low level primitives on the running system. If you have an exemple I'll be very happy to debug it :). But what can surely happen easily is that you can properly stop the container, when you stop the main entrypoint of the container but it's child you start using init system are no longuer proper child of it. You'll certainly need to destroy it in this case that will kill all process inside it.

Will get back shortly with updated patch, but I'll certainly just include systemd into the debian-stretch container.